.pre-commit-config.yaml

```diff
@@ -16,7 +16,7 @@ repos:
       - id: flake8
         additional_dependencies: [flake8-docstrings]
         files: ^edgetest_pip_tools/
-        language_version: python3.9
+        language_version: python3.10
   - repo: https://github.com/jazzband/pip-tools
     rev: 5.5.0
```

Comment:
Should upgrade our pre-commit hook version for pip-tools as it's out of the range in `pyproject.toml`. Does this even do anything with requirements.txt deleted though?

Reply:
Yup good call. I hate having specific versions. Just the latest is fine.

Reply:
Updated the file here with all the latest versions and urls